packages/sui-widget-embedder/downloader/index.js

```diff
@@ -108,11 +108,23 @@
   }
 
   var pages = []
-  for (var key in pathnamesRegExp) {
-    if (window.location.pathname.match(new RegExp(pathnamesRegExp[key]))) {
-      pages.push(key)
+  for (var page in pageConfigs) {
+    var blacklistedRegExps = pageConfigs[page].blacklistedRegExps
```

Comment:
In order to avoid checks with the if, we might want to give a default value.
```
var blacklistedRegExps = pageConfigs[page].blacklistedRegExps || []
```

That would simplify the if a little bit.